doc/specifications/translations/removal.md

```diff
@@ -14,7 +14,7 @@ the existing Versions of a Content Object and is specified as:
  *       a warning before performing it.
  *
  * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified translation
- *         is the only one Content Object has.
+ *         is the only one Content Object has or it is the main translation.
```

Comment:
I think we forgot to discuss & include the case where Content Object Version has more than one translation, but the one being removed is the main translation (`contentInfo->mainLanguageCode`, `versionInfo->initialLanguageCode`). IMHO there is no easy recovery from such situation, so for now I disallowed it throwing `BadStateException`.

It also "covers" the case where a translation is the only one, because the only one translation is the main one (which is reflected in the integration tests PhpDoc).

// cc @andrerom